libpod/container_internal.go

```diff
@@ -683,9 +684,29 @@ func (c *Container) makeBindMounts() error {
 		c.state.BindMounts["/run/.containerenv"] = containerenvPath
 	}
 
+	// Add Secret Mounts
+	c.addSecrets()
+
 	return nil
 }
 
+// addSecrets mounts the secrets from the override and/or default mounts file
+func (c *Container) addSecrets() {
+	mountsFiles := []string{secrets.OverrideMountsFile, secrets.DefaultMountsFile}
```

Comment:
I really doubt we are going to have more than 2 files, so we can drop the array and `for` - use a helper function if you want to clean up

Reply:
fixed